docs/admin/install.rst

```diff
@@ -31,6 +31,9 @@ In order to install SecureDrop Workstation and configure it to use an existing S
 
 - A Qubes-compatible computer with at least 16GB of RAM (32 GB is recommended). SecureDrop Workstation has mainly been tested against Lenovo 6th-gen T480 and X1 models - see Qubes' `Hardware Compatibility List <https://www.qubes-os.org/hcl/>`_ and the SecureDrop Workstation :doc:`hardware` page for more options .
 - Qubes installation medium - this guide assumes the use of a USB 3.0 stick. Qubes may also be installed via optical media, which may make more sense depending on your `security concerns <https://www.qubes-os.org/doc/install-security/>`_.
+
+  .. note:: A USB stick with a Type-A connector is recommended, as USB-C ports may be disabled on your computer when the BIOS settings detailed below are applied.
```

Comment:
IMO if we are borking USB-C we should consider if those BIOS settings are too aggressive instead of considering this a standard troubleshooting issue / hardware restriction to carry forward with us indefinitely.